db/local_db/status.go

```diff
@@ -36,3 +42,41 @@ func GetStatus() (*RunningDBInstanceInfo, error) {
 
 	return info, nil
 }
+
+// errorIfUnknownService errors if it can find a `postmaster.pid` in the `INSTALL_DIR`
+// and the PID recorded in the found `postmaster.pid` is running
+func errorIfUnknownService() error {
+	// no postmaster.pid, we are good
+	if !helpers.FileExists(getPostmasterPidLocation()) {
+		return nil
+	}
+
+	// read the content of the postmaster.pid file
+	fileContent, err := ioutil.ReadFile(getPostmasterPidLocation())
+	if err != nil {
+		return err
+	}
+
+	// the first line contains the PID
+	lines := strings.FieldsFunc(string(fileContent), func(r rune) bool {
+		return r == '\n'
+	})
+	// extract it
+	pid, err := strconv.ParseInt(lines[0], 10, 64)
```

Comment:
is `lines[0]` always non nil - fo we need to check `len(lines)`?

Reply:
fixed